src/components/collectible-details-drawer/CollectibleDetailsDrawer.tsx

```diff
@@ -111,67 +137,104 @@ const CollectibleDetails = () => {
     return url ? getHostname(url) : ''
   }, [collectible])
 
+  const handleShare = useCallback(() => {
+    if (collectible) {
+      const url = `${AUDIUS_URL}/${handle}/collectibles/${getHash(
+        collectible.id
+      )}`
+      share({ url })
+    }
+  }, [handle, collectible])
+
   const styles = useThemedStyles(createStyles)
 
   const ChainLogo = collectible?.chain === Chain.Eth ? LogoEth : LogoSol
+  const buttonIconColor = useColor('neutralLight4')
```

Comment:
No need to change but I prefer to use:

```
const { neutralLight4 } = useThemeColors()
```

because you get type checking on the color name